app/src/main/res/layout/fragment_mediaitem_list.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
nit: Please make sure the `<!--` is on the following line (the formatter loves to move it there x.x)

Reply:
done